### ISSUE TITLE
Verification bullets miss the aggregated-count case, where a right answer from a wrong probe reinforces the probe

### ISSUE THREAD

**Issue by bapinoko-sudo** [rebelytics/one-skill-to-rule-them-all#26]
## Where

`SKILL.md`, the "Verifying prose you wrote yourself" subsection under the Pre-Flight Principle.

## What happened

Putting a personal skill library under local git for the first time, I authored a `.gitignore` and verified it with:

```bash
git status --porcelain --ignored | grep -c '^!!'
# 1
```

I reported "1 ignored". Five files were actually being ignored. Porcelain v1 collapses an ignored *directory* to a single entry, so the count was counting entries and I presented it as a count of files:

```bash
git status --porcelain --ignored --untracked-files=all | grep '^!!'
# !! scripts/__pycache__/core.cpython-312.pyc
# !! scripts/__pycache__/core.cpython-314.pyc
# ...5 total
```

## Why the existing bullets do not catch it

Every bullet in that subsection is keyed on a diagnostic that **contradicts** a strong prior:

- "When a FAIL is reported for content you yourself just wrote, treat the check as the prime suspect before the file."
- and the closing rule: "when a diagnostic contradicts a strong prior, widen the observation rather than sharpen the instrument."

Here the diagnostic **confirmed** the prior. The `.gitignore` was correct, the outcome was fine, and so nothing prompted a second look. The section's trigger condition is effectively "the check disagreed with me", and this failure mode only exists when the check agrees.

The nearest existing bullet, "have every probe state its own scope in its output", covers *truncated* reads (the first-4KB example). It does not cover *aggregated* ones, where the tool is showing you everything it thinks you want, collapsed for readability.

## Suggested change

Two small additions, both of which I have applied locally and which read naturally in the existing voice:

1. A bullet on aggregation:

   > A count is not a listing. Before reporting a number, confirm what its unit actually is, because a tool that aggregates for readability under-reports without saying so. Enumerate and read the set whenever it is cheap enough to enumerate.

2. Extending the closing general rule to the confirming case:

   > The same holds when a diagnostic CONFIRMS a strong prior, which is the harder case, since a right answer from a wrong instrument teaches you to trust the instrument.

Happy to open a PR if that is easier than transcribing.

## Aside, if it fits anywhere

Authoring a `.gitignore` may deserve a sentence of its own, because it inverts the usual signal. An ignore rule governs what will silently fail to be preserved *later*, and its failure mode is a clean `git status`. Over-matching and correct behaviour produce an identical clean tree, so the signal that something is wrong is the same signal that everything is fine. `git check-ignore -v <path>`, which names the rule file and line responsible for a given path, is the probe that actually answers it.

**Comment by bapinoko-sudo** [comment]
Raised as #27.

One correction to the issue above: I located this under a "Verifying prose you wrote yourself" subsection of `SKILL.md`. That was written against an older monolithic version of the file. Current `main` has since split into `SKILL.md` plus `references/`, and that subsection is not in it, so the bullets I proposed extending no longer exist to extend.

The PR is therefore standalone and depends on none of that. It adds a single paragraph to the Pre-Flight Principle in `references/skill-authoring.md`, directly after the embedded-commands rule, since executing a snippet once does not catch this class: the command runs cleanly and returns a plausible number.

**Comment by bapinoko-sudo** [comment]
Correcting my own correction, with apologies for the noise.

I said the "Verifying prose you wrote yourself" subsection came from an older monolithic version of your `SKILL.md`. It did not. Checking my install against every upstream revision of that file, it turns out the subsection was **never** in this repository: it was a local addition to my own copy that I had mistaken for yours. Nothing was removed in the v2.0.0 slimming pass, and there is no need to go looking through the history for it.

The PR (#27) is unaffected, since it was written standalone against current `main` and depends on none of that. Only my description of where the rule came from was wrong.

**Comment by rebelytics** [comment]
Thanks for the details, @bapinoko-sudo. As mentioned on #27, your improvements will be part of v3.

**Comment by rebelytics** [comment]
Resolved by PR #27, shipped in v3.0.0, released today. Thanks again for your contribution — the release notes credit everyone whose reports and PRs shaped this version.
